src/cookbooks/real_time_search.md

```diff
@@ -0,0 +1,24 @@
+
+# See Real-time search metrics
+
+It's easy to get almost real-time search metrics with `telemetry_live` dataset. Below is the query to give the hourly `sap` search metrics per each major search engine since the start of current day in `CA` based on `1%` of the sample. 
```

Comment:
For event pings, it would be reasonable to call the live tables "almost real-time", but I think it's misleading for search metrics. Search metrics are delivered in main pings and often reflect usage over the course of many previous hours, so there is inherent delay.

I would suggest maybe calling this "intraday search metrics".

Reply:
gotcha, updated.